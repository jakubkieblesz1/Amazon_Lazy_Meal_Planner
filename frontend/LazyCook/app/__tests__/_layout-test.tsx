import React from 'react';
import { render } from '@testing-library/react-native';
import Layout from '../_layout';
import { GestureHandlerRootView } from 'react-native-gesture-handler';

jest.mock('expo-router', () => ({
  Stack: jest.fn(() => null),
}));

jest.mock('react-native-gesture-handler', () => ({
  GestureHandlerRootView: jest.fn(({ children }) => children),
}));

describe('Layout Component', () => {
  it('renders GestureHandlerRootView', () => {
    const { toJSON } = render(<Layout />);
    expect(toJSON()).toMatchSnapshot(); // Ensure component tree includes GestureHandlerRootView
  });
});
