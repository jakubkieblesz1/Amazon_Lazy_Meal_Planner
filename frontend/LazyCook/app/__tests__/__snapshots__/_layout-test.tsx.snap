// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Component renders GestureHandlerRootView 1`] = `null`;
